tests/particle_test.cc

```diff
@@ -102,8 +102,7 @@ TEST_CASE("Particle is checked for 1D case", "[particle][1D]") {
     REQUIRE(particle->mass(Phase) == Approx(100.5).epsilon(Tolerance));
 
     // Check stress
-    Eigen::VectorXd stress;
-    stress.resize(Dim);
+    Eigen::Matrix<double, 6, 1> stress;
```

Comment:
I guess we will never have a 1D test for MPM with particles!

Reply:
Easier fix than to remove all tests for 1D particle

Reply:
Maybe removal should be handled in a separate PR?